src/__tests__/__snapshots__/import.spec.ts.snap

```diff
@@ -1,7 +1,27 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`should handle dynamic imports 1`] = `
-"declare type A = $Exports<\\"react\\">;
-declare type B = $PropertyType<$Exports<\\"react\\">, \\"ReactNode\\">;
+"import * as $Flowgen$Import$react from \\"react\\";
+import * as $Flowgen$Import$zlib from \\"zlib\\";
+declare type A = typeof $Flowgen$Import$react;
+declare type B = $Flowgen$Import$react.Node;
+declare type C = $Flowgen$Import$react.ComponentType<{ ... }>;
```

Comment:
Everything looks good to me in the PR, I just want a confirmation on the `...` being legit flow syntax here (because it doesn't live in the test file) and I'm happy to make a build with these changes.

Reply:
Yep! `{ ... }` is the Flow way to spell the type of an object where we don't know about any properties, but it may have arbitrary other properties we don't know about. Docs here:
https://flow.org/en/docs/types/objects/#toc-explicit-inexact-object-types

So that's equivalent to what TypeScript means by `{}`.

In Flow `{}` by default currently means the same thing, but they're [planning](https://flow.org/en/docs/types/objects/#toc-explicit-inexact-object-types) to flip that default so that it means the same as `{||}`, i.e. the type of an object that can't have any properties. There's already an [option](https://flow.org/en/docs/config/options/#toc-exact-by-default-boolean) to choose that future behavior, so I'm glad that Flowgen produces the unambiguous `{ ... }` syntax.

The test also confirms that this is valid Flow syntax, because it calls `expect(…).toBeValidFlowTypeDeclarations()` on the result.